pkg/index/mysql/mysql.go

```diff
@@ -0,0 +1,106 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/index"
+	_ "github.com/lib/pq"
```

Comment:
this import seems superfluous?

Reply:
This is so that the Postgres driver is registered with the `sql/db` standard library. See the package doc here: https://pkg.go.dev/github.com/lib/pq?tab=doc

I will add a comment on top of the import to help with readability a little bit. Thanks for pointing it out 👍

Reply:
Sure, but this is in the `mysql` package? I don't think it needs to be imported in here.

Reply:
Ohh good call. I had both packages under one at some point while I was developing. Will fix it up